src/improvements/tasks/sep/027-U16a-opcm-upgrade-v410-op-soneium-ink/README.md

```diff
@@ -0,0 +1,41 @@
+# 027-U16a-opcm-upgrade-v410-op-soneium-ink: Upgrade 16a: OP, Soneium, Ink on Sepolia
+Status: [DRAFT, NOT READY TO SIGN]()
+
+## Objective
+
+Executes Upgrade 16a on OP, Soneium, and Ink on Sepolia.
+
+This task uses `op-contract/v4.1.0-rc.2` [OPContractsManager](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v4.1.0-rc.2/packages/contracts-bedrock/src/L1/OPContractsManager.sol) to upgrade the following chains:
+
+1. OP Sepolia Testnet
+2. Ink Sepolia Testnet
+3. Soneium Testnet Minato
+
+### Timing
+
+Expected to be executed on or around 2025-09-22.
+
+## Transaction creation
+
+The transaction is generated by the [OPCMUpgradeV410 template script](../../../template/OPCMUpgradeV410.sol),
+which reads the inputs from the [`config.toml`](./config.toml) file.
+
+## Signing and execution
+
+Follow the instructions in the [Nested Execution](../../../NESTED.md) guide for the following steps:
+
+- [1. Update repo](../../../NESTED.md#1-update-repo)
+- [2. Setup Ledger](../../../NESTED.md#2-setup-ledger)
+- [3. Simulate and validate the transaction](../../../NESTED.md#3-simulate-and-validate-the-transaction)
+
+Then follow the instructions in the [Validation](./VALIDATION.md) guide.
+
+## Simulation
+
+When simulating, ensure the logs say `Using script <your_path_to_superchain_ops>/superchain-ops/src/improvements/template/OPCMUpgradeV410.sol`.
+Navigate to the correct task directory then run the simulate command.
+
+```
+cd src/improvements/tasks/sep/027-U16a-opcm-upgrade-v410-op-soneium-ink
+SIMULATE_WITHOUT_LEDGER=1 just --dotenv-path $(pwd)/.env --justfile ../../../../../src/improvements/justfile simulate
```

Comment:
because this is a nested safe, don't you need to pass in `foundation` and then in a separate command `council` to get the correct transaction hashes.